src/apiCalls.js

```diff
@@ -1,19 +1,20 @@
-function getAllData() {
-    const singleTraveler = fetch('http://localhost:3001/api/v1/travelers/20')
-        .then(resp => resp.json())
-
-    const allTrips = fetch('http://localhost:3001/api/v1/trips')
-        .then(resp => resp.json())
+// lets see what we're working with!
+// get that DATA and send it to scripts.js
 
-    const allDestinations = fetch('http://localhost:3001/api/v1/destinations')
-        .then(resp => resp.json())
-
-    Promise.all([singleTraveler, allTrips, allDestinations])
-        .then((data) => {
-            let [singleTraveler, allTrips, allDestinations] = data
-            console.log(singleTraveler, allTrips, allDestinations)
-        })
-        .catch(error => console.log(error))
+function getAllData() {
+    return Promise.all([
+        fetch('http://localhost:3001/api/v1/travelers').then(resp => resp.json()),
+        fetch('http://localhost:3001/api/v1/trips').then(resp => resp.json()),
+        fetch('http://localhost:3001/api/v1/destinations').then(resp => resp.json())
+    ])
+    .then(([travelers, trips, destinations]) => {
```

Comment:
fabulous idea returning all the data in an object for easy and organized access in scripts!

When you move to error handling, checking if the response's `ok` property is `true` and then moving on to parsing might take a little refactoring of lines 6-8, but might be a good place for the safety net that likely won't get used at all